### PR TITLE
Feature: move AutoYaST files to manager-build-profiles

### PR DESCRIPTION
**Pull request by mbologna**
# Description

As discussed in the last review meeting, we want to unify the places where build-related profiles are published. This PR moves those files away from the uyuni-docs repo and publishes them in the manager-build-profiles repo.

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [X] Manager-4.1
- [X] Uyuni

# Links

NOTE:
- This PR already contains changes submitted in https://github.com/uyuni-project/uyuni-docs/pull/321
- The files moved to manager-build-profiles are in the PR: https://github.com/SUSE/manager-build-profiles/pull/15

### CODE REVIEW

**Comment by mbologna** [comment]
CC @cbbayburt @mateiw

**Comment by Loquacity** [comment]
I'm happy to do this for 4.1 forwards, but I don't see a compelling argument to backport to all old versions.

**Comment by keichwa** [comment]
Ok, nice!  But then we want to get rid of example files such as caasp-autoinstallation-example.adoc completely.  Where we include or reference it we now should replace it with the https://github.com/SUSE/manager-build-profiles link.  In this case, maybe even a direct link to the caasp example file.

**Comment by mbologna** [comment]
> I'm happy to do this for 4.1 forwards, but I don't see a compelling argument to backport to all old versions.

OK, I agree: this should go in with 4.1 and Uyuni. If we update the manager-build-profiles, not rebuilt docs (= everything before 4.1) will have an outdated AutoYaST profile.
We will ping you to rebuild those docs in that case.

**Comment by mbologna** [comment]
> Ok, nice! But then we want to get rid of example files such as caasp-autoinstallation-example.adoc completely. Where we include or reference it we now should replace it with the [SUSE/manager-build-profiles](https://github.com/SUSE/manager-build-profiles) link. In this case, maybe even a direct link to the caasp example file.

Sure, I will do it in this PR.

**Comment by keichwa** [comment]
On 2020-06-16 10:38, Michele Bologna wrote:

> Sure, I will do it in this PR.

Thank you, Michele!

-- 
Karl Eichwalder                    SUSE Software Solutions Germany GmbH
                                    Maxfeldstraße 5
                                    90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Comment by Loquacity** [comment]
@mbologna if you can fix the conflict, then i'll merge for you.

File: modules/client-configuration/pages/autoyast-example.adoc [hunk 1]
```diff
@@ -21,56 +22,7 @@ You need to adjust the IP address of the {productname} server, the name of the b
...
-   <script>
-    <interpreter>shell</interpreter>
-    <location>http://192.168.1.1/pub/bootstrap/my_bootstrap.sh</location>
-   </script>
-  </init-scripts>
- </scripts>
-</profile>
-----
-
+You can find the AutoYaST file at https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/Minimal-AutoYaST
```
Comment (keichwa) [inline modules/client-configuration/pages/autoyast-example.adoc:25]:
```suggestion
You can find the AutoYaST file at https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/Minimal-AutoYaST.
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 2]
```diff
@@ -65,7 +65,7 @@ Issuing those operations could render your {caasp} cluster unusable.
 The profile is in the ``patterns-caasp-Management`` package.
 For more information about the profile, see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_autoyast_preparation.
 
-For an example script based based on the {caasp}{nbsp}4 template, customized to make use of {productname}, see xref:client-configuration:caasp-autoinstallation-example.adoc[].
+For an example script based based on the {caasp}{nbsp}4 template, customized to make use of {productname}, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/CaaSP-autoinstall
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc:68]:
```suggestion
For an example script based based on the {caasp}{nbsp}4 template, customized to make use of {productname}, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/CaaSP-autoinstall.
```

File: modules/upgrade/pages/client-x.adoc [hunk 3]
```diff
@@ -81,13 +81,13 @@ For more information, see https://documentation.suse.com/sles/15-SP1/html/SLES-a
 
 Autoinstallation profiles contain all the installation and configuration data needed to install a system.
 They can also contain scripts to be executed after the installation is complete.
-For example scripts that you can use as a starting point, see xref:upgrade:upgrade-script-example.adoc[].
+For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/
```
Comment (keichwa) [inline modules/upgrade/pages/client-x.adoc:84]:
```suggestion
For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST.
```

File: modules/client-configuration/pages/client-automating-preparation.adoc [hunk 4]
```diff
@@ -149,119 +149,9 @@ You can install it using AutoYaST.
...
-    <init-scripts config:type="list">
-      $SNIPPET('spacewalk/minion_script')
-    </init-scripts>
-  </scripts>
-</profile>
-----
+You can use the following example as a starting point.
+Ensure you change the ``<password>`` to match your own settings.
+
+You can find the AutoYaST file at https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/
```
Comment (keichwa) [inline modules/client-configuration/pages/client-automating-preparation.adoc:155]:
```suggestion
You can find the AutoYaST file at https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST.
```

File: modules/upgrade/pages/client-x.adoc [hunk 5]
```diff
@@ -81,13 +81,13 @@ For more information, see https://documentation.suse.com/sles/15-SP1/html/SLES-a
...
 
 
 
 .Procedure: Creating an Autoinstallation Profile
 . In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > Profiles] and upload your autoinstallation profile script.
-For example scripts that you can use as a starting point, see xref:upgrade:upgrade-script-example.adoc[].
+For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/
```
Comment (keichwa) [inline modules/upgrade/pages/client-x.adoc:90]:
```suggestion
For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST.
```

File: modules/upgrade/pages/client-y-z.adoc [hunk 6]
```diff
@@ -45,7 +45,7 @@ For more information, see xref:client-configuration:client-automating-preparatio
 . You can use variables in the profile to distinguish between the different lifecycle environments.
 
 
-For example autoinstallation profiles, see xref:upgrade:upgrade-script-example.adoc[].
+For example autoinstallation profiles, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/
```
Comment (keichwa) [inline modules/upgrade/pages/client-y-z.adoc:48]:
```suggestion
For example autoinstallation profiles, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST.
```

File: modules/upgrade/pages/proxy-x.adoc [hunk 7]
```diff
@@ -69,13 +69,13 @@ When you have created an autoinstallable distribution, you can edit it by naviga
 
 Autoinstallation profiles contain all the installation and configuration data needed to install a system.
 They can also contain scripts to be executed after the installation is complete.
-For example scripts that you can use as a starting point, see xref:upgrade:upgrade-script-example.adoc[].
+For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/
```
Comment (keichwa) [inline modules/upgrade/pages/proxy-x.adoc:72]:
```suggestion
For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST.
```

File: modules/upgrade/pages/proxy-x.adoc [hunk 8]
```diff
@@ -69,13 +69,13 @@ When you have created an autoinstallable distribution, you can edit it by naviga
...
 
 
 
 .Procedure: Creating an Autoinstallation Profile
 . In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > Profiles] and upload your autoinstallation profile script.
-For example scripts that you can use as a starting point, see xref:upgrade:upgrade-script-example.adoc[].
+For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST/
```
Comment (keichwa) [inline modules/upgrade/pages/proxy-x.adoc:78]:
```suggestion
For example scripts that you can use as a starting point, see https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST.
```

**Comment by mbologna** [comment]
LGTM

**Comment by Loquacity** [comment]
This one might need to be recreated, sorry @mateiw :(

**Comment by mbologna** [comment]
Rebased on top of master

**Comment by Loquacity** [comment]
nice git-fu! Merging ...

**Pull request merged**